src/main/c/cext/ruby.c

```diff
@@ -4123,7 +4123,7 @@ int rb_eql(VALUE obj1, VALUE obj2) {
 }
 
 VALUE rb_obj_clone(VALUE obj) {
-  rb_tr_error("rb_obj_clone not implemented");
+  return truffle_invoke(RUBY_CEXT, "rb_obj_clone", obj);
```

Comment:
We can do the call directly in such a case, no need for a helper method.
But maybe it has advantages for debugging?

Reply:
I have no preference one way or the other. I thought I was sticking with the established pattern.